src/instructlab/model/backends/vllm.py

```diff
@@ -120,32 +164,56 @@ def run_vllm(
         background (bool):            Whether the stdout and stderr vLLM should be sent to /dev/null (True)
                                       or stay in the foreground(False).
     Returns:
-        vllm_process (subprocess.Popen): process of the vllm server
+        tuple: A tuple containing two values:
+            vllm_process (subprocess.Popen): process of the vllm server
+            tmp_files: a list of temporary files necessary to launch the process
+
     """
     vllm_process = None
     vllm_cmd = [sys.executable, "-m", "vllm.entrypoints.openai.api_server"]
+    tmp_files = []
```

Comment:
What's the purpose of having a list here? From what I can tell `tmp_files` is either empty or contains one entry.

Reply:
Its's more for the future, since any passing of a large value or hard to parse value on the prompt we would need to store to the fs